CMP1903_A1_2324/Testing.cs

```diff
@@ -15,6 +16,20 @@ internal class Testing
          * Use debug.assert() to make the comparisons and tests.
          */
 
-        //Method
+        public void test() //Method
+        {
+            Game TestGame = new Game();
+
+            for (int i = 0; i < 100; i++)
+            {
+                var TestGameResult = TestGame.StartGame();
+
+                Debug.Assert(TestGameResult.Item1 >= 3 && TestGameResult.Item1 <= 18, "Sum Value out of range");
+                Debug.Assert(TestGameResult.Item2 >= 1 && TestGameResult.Item2 <= 6, "Die Value out of range");
+                Debug.Assert(TestGameResult.Item3 >= 1 && TestGameResult.Item3 <= 6, "Die Value out of range");
+                Debug.Assert(TestGameResult.Item4 >= 1 && TestGameResult.Item4 <= 6, "Die Value out of range");
```

Comment:
Error handlers work well, boundary values are correct and stop any anomalous values

Errors handlers could be separated into: "value to large" or "value to small". The dice number where the error is contained could be outputted to be more descriptive of the location of the error.